daemon/daemon.go

```diff
@@ -363,14 +376,16 @@ func (daemon *Daemon) restore() error {
 			}
 
 			if c.IsRunning() || c.IsPaused() {
+				logger(c).Debug("Syncing container on disk state with real state")
```

Comment:
nit: perhaps lowercase to be consistent with other logs?

```suggestion
				logger(c).Debug("syncing container state on disk with actual state")
```